tests/test_manage.py

```diff
@@ -143,17 +168,33 @@ def test_create_duplicate_request(client):
     assert res.status_code == 201, res.text
 
 
-def test_create_request_without_access(client, mock_arborist_requests):
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            # request with policy_id
+            "username": "requestor_user",
+            "policy_id": "test-policy",
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+        {
+            # request with role_ids and resource_paths
+            "username": "requestor_user",
+            "role_ids": ["study_registrant"],
+            "resource_paths": ["/study/123456"],
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+    ],
```

Comment:
Maybe this makes more sense in [this function](https://github.com/uc-cdis/requestor/blob/7655eba0437f6fd9bf0997cae8baedbc85038b14/tests/test_manage_resource_path.py#L114-L114). When we introduced `policy_id` in addition to `resource_path`, we split the unit tests into two parts, one with `policy_id` and another without. So wasn't sure if this case comes here or there.